src/style.css

```diff
@@ -1,3 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.logo_link {
+  height: 40px;
+}
+
 body {
   margin: 0;
 }
+
+header {
```

Comment:
Avoid using tag names for styling. It's better to use class names instead. (Checklist: Don't use tag names for styling)